.github/workflows/run_functional_product_tests.yaml

```diff
@@ -0,0 +1,56 @@
+# Author: James Garriss
+# Purpose: Run the product functional tests every night
+
+name: Nightly Product Functional Tests
+
+# Run this workflow at midnight 
+# on every Sun to Thr (b/c GMT -5)
+on:
+  push:
```

Comment:
I assume we do not want this to trigger on push.  Is there a need for this?

Reply:
You are correct.  This was added back in order to make some changes requested by Addam.  It will be removed.